modules/backend/classes/FormField.php

```diff
@@ -329,7 +334,9 @@ protected function evalConfig($config)
         if (isset($config['containerAttributes'])) {
             $this->attributes($config['containerAttributes'], 'container');
         }
-
+        if (isset($config['inlineValidation'])) {
+            $this->inlineValidation = boolval($config['inlineValidation']);
```

Comment:
why `boolval` instead of just typecasting to boolean with `(bool) $config['inlineValidation']`?

Reply:
No particular reason - just the first function that came to mind.